### PR TITLE
Ann majeske greenfield nh

### LINKED ISSUE #6: githubteacher's things to do
- [x] Create an issue
- [x] Create a branch
- [x] Create commits
- [x] Create a PR
- [ ] Collaborate!
- [ ] Merge!!! 🚢

### PR DESCRIPTION
**Pull request by AnnMajeske**
This pull request adds a file with things to do and places to eat in Greenfield NH.  This PR resolves #6.

### CODE REVIEW

File: AnnMajeske-hometown.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Greenfield NH
+
+### Things to do: not much
+
+### Places to eat:  not many
```
Comment (AnnMajeske) [inline AnnMajeske-hometown.md]: Comment on places to eat line

File: AnnMajeske-hometown.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Greenfield NH
+
+### Things to do: not much
```
Comment (AnnMajeske) [inline AnnMajeske-hometown.md]: Comment on Things to do line

Comment (myersb-dell) [inline AnnMajeske-hometown.md]: Sounds like an exciting place to live!

File: AnnMajeske-hometown.md (hunk 1 above)
Comment (myersb-dell) [inline AnnMajeske-hometown.md]: Any good burger places?

File: AnnMajeske-hometown.md (hunk 2 above)
Comment (AnnMajeske) [inline AnnMajeske-hometown.md]: LOL

**Pull request merged**